figures/scripts/figS2-snv-callers-panels.R

```diff
@@ -185,17 +203,19 @@ for (dataset in c("tcga", "pbta")) {
 
   ## VAF correlation plots --------------------------------------------
   print("Making VAF correlation plot")
-  # Correlate VAFs across callers pbta
-  cor_vaf <- all_caller_df %>%
-    select(starts_with("VAF_")) %>%
-    # 1:n() column for spread/gather bookkeeping which we do to remove `VAF_` from strips in this plot
-    mutate(n = 1:n()) %>%
-    # we want to remove
-    gather(caller, vaf, -n) %>%
-    mutate(caller = str_replace_all(caller, "VAF_", "")) %>%
-    spread(caller, vaf) %>%
-    select(-n) %>%
-    GGally::ggpairs(aes(alpha = 0.05)) +
+  # Correlate VAFs across callers
+  detect_mat_df <- as.data.frame(detect_mat) %>%
```

Comment:
I'm thinking maybe rename this for clarity?
```suggestion
  vaf_mat_df <- as.data.frame(vaf_mat) %>%
```